config/initializers/sidekiq.rb

```diff
@@ -16,12 +19,6 @@ def not_past_scheduled_time?(current_time)
   end
 end
 
-Rails.application.config.to_prepare do
-  Dir.glob(Rails.root.join("lib/sidekiq/*.rb")).each do |filename|
-    require_dependency filename
-  end
-end
-
```

Comment:
Looks like [`require_dependency` was removed](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#require-dependency), so this change seems reasonable, but should we check to see whether it will reload on change?

Asking since `require_dependency` added that file to the list of files that Rails processed for hot code reloading.